internal/deprecations.go

```diff
@@ -97,6 +97,39 @@ func NormalizeClusterSpec(cluster *fdbv1beta2.FoundationDBCluster, options Depre
 		updateImageConfigs(&cluster.Spec, cluster.GetUseUnifiedImage())
 	}
 
+	if len(cluster.Spec.Buggify.CrashLoop) > 0 {
```

Comment:
Can we make sure we have unit tests for this conversion to ensure it behaves as expected?

Reply:
I think, the existing CrashLoop tests should pass if this is implemented correctly which should verify if the behavior is preserved.

Reply:
I'm okay with that personally I would prefer to have a dedicated that ensures the conversions works in all cases e.g.:

- CrashLoop is empty and CrashLoopContainers is empty
- CrashLoop is not empty and CrashLoopContainers is empty
- CrashLoop is not empty and CrashLoopContainers is not empty

I believe the existing tests only cover `CrashLoop is not empty and CrashLoopContainers is empty`.

Reply:
Existing CrashLoop tests will cover "CrashLoop empty/not-empty and CrashLoopContainer empty", as the behavior expected by those tests are fulfilled by the conversions. The only combination left is CrashLoopContainer empty.